sources/packages/api/src/route-controllers/institution/institution.aest.controller.ts

```diff
@@ -228,4 +232,21 @@ export class InstitutionAESTController extends BaseController {
       id: institution.id,
     };
   }
+
+  /**
+   * Create a user, associate with the institution, and assign the authorizations.
+   * @param payload authorizations to be associated with the user.
+   * @returns Primary identifier of the created resource.
+   */
+  // TODO: Add API Responses and validations. This method will be worked in the upcoming PR.
+  @Post(":institutionId/user")
+  async createInstitutionUserWithAuth(
+    institutionId: number,
```

Comment:
Pls, add `institutionId` in the comment.

Reply:
I will add it.

Reply:
Comment added.